src/SplitPackages/project.json

```diff
@@ -16,6 +16,11 @@
         "NuGet.Client": "3.4.0-*"
     },
     "frameworks": {
+        "dnxcore50": {
+            "dependencies": {
+                "NETStandard.Library": "1.0.0-rc2-*"
```

Comment:
We mostly spent time on `dev` and `rel/1.0.0` branch where we use `rc2` packages only. More tests are done on `rc2` packages.